measure_extinction/extdata.py

```diff
@@ -702,22 +698,22 @@ def trans_elv_alav(self, av=None, akav=0.112):
 
     def rebin_constres(self, source, waverange, resolution):
         """
-        Rebin the source extinction curve it a fixed spectral resolution
+        Rebin the source extinction curve to a fixed spectral resolution
         and min/max wavelength range.
 
         Parameters
         ----------
         source : str
             source of extinction (i.e. "IUE", "IRS")
         waverange : [float, float]
```

Comment:
I guess technically the values in waverange are then astropy quantities instead of floats. I know, I am nitpicking now.